scene/gui/item_list.cpp

```diff
@@ -1383,8 +1394,10 @@ void ItemList::force_update_list_size() {
 		max_column_width = MAX(max_column_width, minsize.x);
 
 		// elements need to adapt to the selected size
```

Comment:
(While you're here and decided to add extra newlines)
```suggestion
		// Elements need to adapt to the selected size.
```